<?xml version="1.0"?>
<launch>
	
	<!-- general arm components fir inverse kinematic -->
    <include file="$(find youbot_arm_navigation)/ros/launch/planning_environment.launch" />

	<include file="$(find youbot_arm_navigation)/ros/launch/environment_server.launch">
		<arg name="use_monitor" value="true" />
		<arg name="use_collision_map" value="false" />
	</include>

	<include file="$(find youbot_arm_navigation)/ros/launch/constraint_aware_kinematics.launch" >
        <arg name="use_monitor" value="true" />
		<arg name="use_collision_map" value="false" />
	</include>
	<include file="$(find edufill_youbot_arm_geometrical_solution)/launch/arm_geometry_solution.launch" />

    <node pkg="hbrs_arm_cart_control" type="arm_cart_control" ns="arm_1" name="arm_cart_control" output="screen" respawn="false">
      <param name="root_name" value="arm_link_0"/>
      <param name="tip_name" value="arm_link_5"/>
      <remap from="~joint_velocity_command" to="arm_controller/velocity_command"/>
    </node>

</launch>
